src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+function Person({ person }) {
+  return (
+    <>
+      <section className="Person">
+        <h2 className="Person__name">
+          {`My name is ${person.name}`}
+        </h2>
+
+        {person.age && (
+        <p className="Person__age">
+          {`I am ${person.age}`}
+        </p>
```

Comment:
don't forget about code style, add 2 spaces
```suggestion
          <p className="Person__age">
            {`I am ${person.age}`}
          </p>
```